src/App.tsx

```diff
@@ -143,20 +147,50 @@ const Test = connect(
         onLearn: (item: model.Learnable) => dispatch(actions.learn(item)),
         onReview: (id: model.LearnableId, correct: boolean) =>
             dispatch(actions.review(id, correct)),
-        onWander: () => dispatch(actions.wander()),
+        onWander: () => {
```

Comment:
Why not just create a `modifyResource` action dispatcher, then make `onMeditate` and `onWander` methods of `App`? That way you can pass `modifyResource` to other components that find it useful.

Reply:
addressed in 8410c77